ResidentContactApi.Tests/V1/Controllers/ResidentContactApiControllerTest.cs

```diff
@@ -180,5 +184,49 @@ public void InsertResidentReturns500StatusCodeIfExternalReferenceNotInsertExcept
             result.StatusCode.Should().Be(500);
             result.Value.Should().Be("External reference could not be inserted - error message");
         }
+
+        [Test]
+        public void IfExternalResidentToBeInsertedAlreadyExistsShouldReturn200StatusCode()
```

Comment:
Is this testing for if the resident record exists or if the external reference exists?